collect_app/src/main/java/org/odk/collect/android/utilities/FormDownloader.java

```diff
@@ -341,8 +345,7 @@ private Uri saveNewForm(Map<String, String> formInfo, File formFile, String medi
      * Takes the formName and the URL and attempts to download the specified file. Returns a file
      * object representing the downloaded file.
      */
-    FileResult downloadXform(String formName, String url)
-            throws IOException, TaskCancelledException, Exception {
+    FileResult downloadXform(String formName, String url) throws Exception {
```

Comment:
This is not ideal but fine for now. It'd be better to actually enumerate the specific exceptions that can occur but `downloadFile` throws a generic `Exception` so we can't.